### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by MickVermeulen**
Added issue templates to aid with the structural submission of issues.

### CODE REVIEW

**Review by MasterEric — APPROVED** [review]
Looks great! Will definitely help keep stuff organized

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+
+name: Feature Request
+about: Suggest an idea for DarkflameServer
+title: "ENH:"
+labels: "Enhancement, Needs Triage"
+
+---
+
+#### Is your feature request related to a problem?
```
Comment (luxaritas) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]: Is there a reason that only this template doesn't use the fancy fields, but just uses raw markdown instead?

Comment (MickVermeulen) [reply .github/ISSUE_TEMPLATE/feature_request.md:10]: No not really, will actually change that :)

Comment (MickVermeulen) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Updated to also make use of the yaml syntax.

**Pull request merged**